docs/ehrql-new-tutorial-conclusion.md

```diff
@@ -62,6 +62,15 @@ as reference for writing your own dataset definitions:
 !!! todo
     Can we point to real, working Data Builder examples here?
 
+!!! tip
+    A good way to learn ehrQL is to experiment
+    by trying out ehrQL features against a simple fake dataset.
```

Comment:
add link to the https://docs.opensafely.org/ehrql-reference/ here?

Reply:
Done!